sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/query.sql

```diff
@@ -100,7 +125,7 @@ count_picked AS (
     COALESCE(SUM(position.value), 0) AS count_picked_total,
     mozfun.map.sum(ARRAY_AGG(STRUCT(type AS key, position.value AS value))) AS count_picked_by_type,
     mozfun.map.sum(
-      ARRAY_AGG(STRUCT(one_index(position.key) AS key, position.value AS value))
+      ARRAY_AGG(STRUCT(position.key AS key, position.value AS value))
```

Comment:
I noticed this while writing tests as well, it seems like this call was causing all the index -> position conversions to be done twice, since it's already done above. So 0 -> 2 instead of 0 -> 1. Let me know if I'm misunderstanding though @teonbrooks

Reply:
nice catch. removing it looks right to me

Reply:
nice catch. agreed!